addons/website_sale_autocomplete/controllers/main.py

```diff
@@ -20,6 +20,17 @@
     'administrative_area_level_2': ['state', 'country']
 }
 
+CHILEAN_STATES = {
+    'Araucanía': 'de la Araucanía',
+    'Región Metropolitana': 'Metropolitana',
+    'Bío Bío': 'del BíoBío',
```

Comment:
in the XML there is an accent on the first i but not on the second

```suggestion
    'Bío Bío': 'del BíoBio',
```